server-common/src/main/java/io/a2a/server/requesthandlers/RequestHandler.java

```diff
@@ -14,13 +14,18 @@
 import io.a2a.spec.Task;
 import io.a2a.spec.TaskIdParams;
 import io.a2a.spec.TaskPushNotificationConfig;
+import io.a2a.spec.TaskListParams;
 import io.a2a.spec.TaskQueryParams;
 
 public interface RequestHandler {
     Task onGetTask(
             TaskQueryParams params,
             ServerCallContext context) throws JSONRPCError;
 
+    List<Task> onListTasks(
```

Comment:
Looks like Stefano's PR to remove this was merged and there's a comment there saying this is not intended to be part of the 0.3.0 spec:

https://github.com/a2aproject/A2A/pull/990#issuecomment-3198035468

There's a separate PR to add it but it's still under review:

https://github.com/a2aproject/A2A/pull/831

So I think we can remove this here for now until it's incorporated into a spec version.

Reply:
+1